### ISSUE TITLE
BUG: Controls locking up in Top Gun - Hard Lock [BLUS30858]

### ISSUE THREAD

**Issue by Bobble80** [RPCS3/rpcs3#15029]
### Quick summary

There seems to be some input and controller issues with this game. The game boots and runs fine, but when you get to the title screen and menus, the circle input button doesn't seem to be working. 
Then when you start the game and get to the first main mission, the plane infinitely pitches itself upwards, making the game unplayable.

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/13875983/RPCS3.log.gz)

**Comment by RipleyTom** [comment]
You appear to be using SDL as Input Handler. Since you have an xbox 360 pad I'd suggest trying XInput before anything else.

**Comment by Bobble80** [comment]
> You appear to be using SDL as Input Handler. Since you have an xbox 360 pad I'd suggest trying XInput before anything else.

The app controller config just defaulted to SDL, that seemed to work for everything from the start. I’m using an ROG Ally with its built in gamepad, as far as I know windows names it as an Xbox 360 pad in the settings. I’ve tested it with the built in Ally controls and also an external Bluetooth Xbox controller, same issue with both. 

I tried to change the input to xinput anyway and tested it again, made no difference. The issue seems to be with the game itself. Any other games I’ve tried work out of the box with the default controller settings.

**Comment by meisme-dev** [comment]
I have this issue on Linux with EVDEV

**Comment by Littlemouse1968** [comment]
To resolve the controller issue, change the Mouse Handler option in the I/O View from Basic to Null.

**Comment by digant73** [comment]
@Bobble80 is this issue still present? also did you try @Littlemouse1968's suggestion?

**Comment by Bobble80** [comment]
> [@Bobble80](https://github.com/Bobble80) is this issue still present? also did you try [@Littlemouse1968](https://github.com/Littlemouse1968)'s suggestion?

No, it’s fixed by changing the mouse handler to null like what was suggested. 
It can be noted in the wiki now and game can be changed to playable.

**Comment by digant73** [comment]
Ok. I don't know who can update the wiki page. Maybe @AniLeo

**Comment by AniLeo** [comment]
Perhaps mouse handler should be null by default, since you won't be having a mouse connected while playing games on a PS3 too

@Megamouse for input

**Comment by elad335** [comment]
iirc it was set to on by default as a hack for Dark Souls 2 booting, need to to test if it still needs it.

**Comment by digant73** [comment]
apparently fixed but it requires an input from @Megamouse and possibly an update on wiki page

**Comment by GorgeousGorillawitz** [comment]
Just to note, Move handler also needs to be null. Changing mouse handler to null alone didn't fix it for me, but also Move handler to null did.

**Comment by ShadowH69** [comment]
Forgot about this issue , setting the mouse handler to Null is more than enough , this issue should be closed as the game is now in playable and confirmed working , the wiki was already updated to accommodate this.
